Sources/SaferContinuation/SaferContinuationError.swift

```diff
@@ -1,15 +1,15 @@
 import Foundation
 
 public enum SafeContinuationError: Error {
-	/// internally used by safercontinuation. cannot be thrown by the continuation as that would cause bugs to have the continuation resume multiple times (which is exactly what we are avoiding here)
-	case alreadyRun(file: StaticString, line: Int, function: StaticString, context: Any?)
+    /// internally used by safercontinuation. cannot be thrown by the continuation as that would cause bugs to have the continuation resume multiple times (which is exactly what we are avoiding here)
+    case alreadyRun(file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil)
 
-	/// internally used by safercontinuation. cannot be thrown by the continuation as that would cause bugs to have the continuation resume multiple times (which is exactly what we are avoiding here)
-	case previouslyRanButUnreleased(file: StaticString, line: Int, function: StaticString, context: Any?)
+    /// internally used by safercontinuation. cannot be thrown by the continuation as that would cause bugs to have the continuation resume multiple times (which is exactly what we are avoiding here)
+    case previouslyRanButUnreleased(file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil)
 
-	/// thrown by safer continuation when the continuation is deinited and has never been called
-	case continuationNeverCompleted(file: StaticString, line: Int, function: StaticString, context: Any?)
+    /// thrown by safer continuation when the continuation is deinited and has never been called
+    case continuationNeverCompleted(file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil)
 
-	/// thrown by safer continuation when the continuation doesn't fire within the timeout period
-	case timeoutMet(file: StaticString, line: Int, function: StaticString, context: Any?)
+    /// thrown by safer continuation when the continuation doesn't fire within the timeout period
+    case timeoutMet(file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil)
```

Comment:
I meant they should be tabs, not spaces.

However, I now remember why I hadn't done this in the first place: these values are not about the file/line/function/context of where the error is thrown, but about where the continuation is created, hence why the values are forwarded from the continuation properties. It was correct to not have default values in the first place.

Reply:
The code indentation is something I missed.

These default values are useful. When we use these errors outside, at our real code, these default values are really convenient. When use these errors inside this project, we should pass the value from outside.

Reply:
These errors should only be initialized internally. External methods will receive them when thrown, but shouldn't be creating them. And again, the metadata of the file/line/function is to help identify which *continuation* is having issues, not the error itself... So the default arguments to get the error's location is irrelevant, regardless.

Reply:
Done.